docs/UserGuide.md

```diff
@@ -273,6 +346,22 @@ _Details coming soon ..._
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Glossary
+The definitions in this glossary are context-specific to this application.
+
+Word | Definition
+--------|------------------
+**Command Line Interface (CLI)** | A text-based interface that recieves typed commands as input and returns textual feedback as output.
+**Graphical User Interface (GUI)** | An image-based interface that is more visually appealing than a command-line interface and encapsulates information through the use icons and images.
+**Loan** | An amount of money that is borrowed by or owed to a person. A positive value signifies an amount owed by the person and a negative value signifies an amount to be paid to that person.
+**Parameter** | A value passed as a section of a command, typically following a prefix.
+**Prefix** | A signposting word that indicates the kind of property (i.e. name, email, address, etc), which typically follows immediately after the prefix, that is to be passed as a parameter.
+**Property** | An identifiable feature a person or object has that sufficiently distinguishes it from other objects of the same kind.
+**Secretary** | A person that manages the tasks and events related to the operations of an organisation.
+**Tag** | A label that connects two or more related people together, such that they can be referred to as a group specified by the tag.
```

Comment:
Is this the definition we are giving tag? seems like its restricting the multiplicity to 2..*